webapp/store/ui/index.js

```diff
@@ -0,0 +1,5 @@
+import * as UiActions from './actions'
+import UiReducer from './reducer'
+import * as UiState from './state'
+
+export { UiActions, UiReducer, UiState }
```

Comment:
we don't have to export UIActions, only HomeActions, HomeReducer etc..
otherwise ui might become huge. Home is under ui